config.json

```diff
@@ -1,296 +1,278 @@
+                "slug": "triangle",
+                "name": "Triangle"
+            },
+            {
+                "topics": null,
+                "difficulty": 2,
+                "uuid": "9bac9221-a7dc-4c8b-9df3-511652dc2bac",
+                "practices": [],
+                "prerequisites": [],
+                "slug": "word-count",
+                "name": "Word Count"
+            },
+            {
+                "topics": null,
+                "difficulty": 3,
+                "uuid": "06580c1b-5c49-4185-b0ad-fb813bb8e1af",
+                "practices": [],
+                "prerequisites": [],
+                "slug": "beer-song",
+                "name": "Beer Song"
+            },
+            {
+                "topics": null,
+                "difficulty": 3,
+                "uuid": "7be24221-22b0-4741-bf84-4c470242a388",
+                "practices": [],
+                "prerequisites": [],
+                "slug": "isbn-verifier",
+                "name": "Isbn Verifier"
+            },
+            {
+                "topics": null,
+                "difficulty": 3,
+                "uuid": "ecdaf7a8-91dc-489e-a3c6-c111dca481dc",
+                "practices": [],
+                "prerequisites": [],
+                "slug": "leap",
+                "name": "Leap"
+            },
+            {
+                "topics": null,
+                "difficulty": 3,
+                "uuid": "fd8d11f2-f232-4ce1-ae50-234f1306c45e",
+                "practices": [],
+                "prerequisites": [],
+                "slug": "phone-number",
+                "name": "Phone Number"
+            },
+            {
+                "topics": [
+                    "recursivity"
+                ],
+                "difficulty": 4,
+                "uuid": "25e84a52-3231-4a58-a24a-8a20e3736443",
+                "practices": [],
+                "prerequisites": [],
+                "slug": "spiral-matrix",
+                "name": "Spiral Matrix"
+            },
+            {
+                "topics": null,
+                "difficulty": 5,
+                "uuid": "60e7564d-af0d-4be8-9475-05f0f6f246a4",
+                "practices": [],
+                "prerequisites": [],
+                "slug": "clock",
+                "name": "Clock"
+            },
+            {
+                "slug": "allergies",
+                "name": "Allergies",
+                "uuid": "0793ac96-5280-4f0d-a08a-aa57ad9b9946",
+                "practices": [],
+                "prerequisites": [],
+                "difficulty": 4,
+                "topics": null
+            },
+            {
+                "slug": "dominoes",
+                "name": "Dominoes",
+                "uuid": "1c44cacd-6a2d-4f9a-813c-2e1e59a7fe0d",
+                "practices": [],
+                "prerequisites": ["numbers", "booleans"],
+                "difficulty": 4,
+                "topics": [
+                    "graph_theory", "games"
+                ]
```

Comment:
You don't need to add topics for new exercise, as topics is a deprecated key. The only reason we didn't remove it from each track's config.json is because it can help populate the `practices` and/or `prerequisites` fields. Once those are populated, the `topics` property can be deleted (it can already be deleted when it's value is empty or `null`).